api/views.py

```diff
@@ -159,19 +148,17 @@ def stand(request, id):
         elif dealer_score <= 16:
             card_dealt = blackjack_game.deal()
             dealer_hand.append(card_dealt)
-            print(dealer_hand)
-            # I don't think I need to do this until the game is over...
+
             db_game.dealer_hand = json.dumps(dealer_hand)
```

Comment:
i dont think we want to `save` and serialize the dealer hand every time the dealer draws a card. can we move this to the end of the function?